scss/grid/_grid.scss

```diff
@@ -30,6 +30,11 @@ $grid-column-align-edge: true !default;
 /// @type Number
 $block-grid-max: 8 !default;
 
+/// Selector used for an alias of column (with @extend). If `false`, no alias is created.
+/// @since 6.4.0
```

Comment:
That is interesting. If we add support for `@since` in the docs generator it would be an easier to support alternative to having docs versions for every minor release, but might still provide much of that value. @rafibomb what do you think?